Solutions/Jamf Protect/Analytic Rules/JamfProtectAlerts.yaml

```diff
@@ -0,0 +1,81 @@
+id: 6098daa0-f05e-44d5-b5a0-913e63ba3179
+name: Jamf Protect - Alerts 
+description: |
+  'Creates an incident based on Jamf Protect Alert data in Microsoft Sentinel'
+severity: High
+status: Available
+requiredDataConnectors:
+  - connectorId: JamfProtect
+    dataTypes: 
+      - jamfprotect_CL
+suppressionDuration: PT5H
+suppressionEnabled: false
+tactics:
+techniques:
+alertRuleTemplateName:
+query: |
+  jamfprotect_CL
+  | where topicType_s == "alert"
+  | where input_eventType_s <> "GPUnifiedLogEvent"
+  | extend severity = case(input_match_severity_d == 0, "Informational", input_match_severity_d == 1, "Low", input_match_severity_d == 2, "Medium", input_match_severity_d == 3, "High", "True")
+  | extend Analytic_description =  parse_json(input_match_facts_s)[0].human
```

Comment:
Field input_match_facts_s is getting parsed multiple times. Please parse this field once and use the resultant dynamic object to get the various values.